Microsoft.Dotnet.Wpf.sln

```diff
@@ -1,562 +1,632 @@
-﻿
-Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
```

Comment:
Can this be re-done to limit changes to what is needed and no more ?

Reply:
I think it's probably best to do the config changes as a quick follow-up PR so we don't get the noisy diff.

Reply:
It should be good now.